osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -96,60 +104,78 @@ public BreakOverlay(bool letterboxing)
                         Anchor = Anchor.Centre,
                         Origin = Anchor.Centre,
                     }
-                }
-            };
-        }
-
-        protected override void LoadComplete()
-        {
-            base.LoadComplete();
-            initializeBreaks();
-        }
-
-        private void initializeBreaks()
-        {
-            if (!IsLoaded) return; // we need a clock.
-
-            FinishTransforms(true);
-            Scheduler.CancelDelayedTasks();
-
-            if (breaks == null) return; //we need breaks.
-
-            foreach (var b in breaks)
-            {
-                if (!b.HasEffect)
-                    continue;
+          }
+      };
+    }
 
-                using (BeginAbsoluteSequence(b.StartTime, true))
-                {
-                    fadeContainer.FadeIn(fade_duration);
-                    breakArrows.Show(fade_duration);
+    protected override void LoadComplete()
```

Comment:
Ordering `LoadComplete()` below `load()` would be preferrable. Also spaces between methods.

Reply:
Fixed